cmd/service-controller/console_server.go

```diff
@@ -537,6 +537,7 @@ func (services HttpServiceStatsMap) updateHttpRequestStats(siteId string, reques
 type Site struct {
```

Comment:
low-lying but might Site and SiteInfo from site_query be coalesced...

Reply:
Yes, they could actually. I am planning an upcoming fix to the console server to get the ip translation in the right phase, and will merge them at that time.